<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsOutput</class>
 <widget class="QWidget" name="PrefsOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_main">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_options">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="_font">
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="_style">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ShowBold">
              <property name="text">
               <string>Bold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ShowItalic">
              <property name="text">
               <string>Italic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ShowUnderline">
              <property name="text">
               <string>Underline</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="_new_activity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Sound to play on new activity</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="NewActivitySound_test">
            <property name="text">
             <string>Test</string>
            </property>
            <property name="icon">
             <iconset theme="audio-volume-high-symbolic"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="NewActivitySound_none">
            <property name="text">
             <string>No sound</string>
            </property>
            <property name="icon">
             <iconset theme="user-trash-symbolic"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="NewActivitySound_browse">
            <property name="text">
             <string>Browse...</string>
            </property>
            <property name="icon">
             <iconset theme="document-open"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NewActivitySound"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="_checkboxes">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="DisableCompression">
           <property name="text">
            <string>Disable compression</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="IndentParas">
           <property name="text">
            <string>&amp;Indent paragraphs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Naws">
           <property name="text">
            <string>Negotiate about window size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CarriageReturnClearsLine">
           <property name="text">
            <string>Carriage-return clears line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Utf8">
           <property name="text">
            <string>UTF-&amp;8 (Unicode)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ConvertGaToNewline">
           <property name="text">
            <string>Convert IAC EOR/GA to newline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>NewActivitySound_none</sender>
   <signal>clicked()</signal>
   <receiver>NewActivitySound</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
